docs/data-sources/lan.md

```diff
@@ -14,16 +14,18 @@ The lans data source can be used to search for and return existing lans.
 
 ```hcl
 data "ionoscloud_lan" "lan_example" {
-  name     = "My Lan"
+  datacenter_id = ionoscloud_datacenter.example.id
+  name			= "example_"
```

Comment:
I think it would be great, that at some point, to start using `gofmt` for code formatting. Right now, it would probably be a lot of changes to add and it is not a priority.